cmd/kpod/version.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"os/exec"
+	"text/template"
+	"github.com/urfave/cli"
+	"runtime"
+)
+
+//Function to get info for version command
+func getVersion() {
+
+	//Template for the version
+	var versionTemplate = `Client:
+ Version:	{{.Version}}
+ Go Version:	{{.GoVersion}}
+ Git Commit:	{{.GitCommit}}/{{.Version}}
+ Built:		{{.BuildInfo}}
+ OS/Arch:	{{.Os}}/{{.Arch -}}
+ `
+
+ 	var (
+		cmdOut []byte
+		err    error
+	)
+
+	type versionInfo struct {
+	 	Version, GoVersion, GitCommit, BuildInfo, Os, Arch string
+	}
+
+	var info = versionInfo {}
+
+	info.Version = VERSION
+	info.GoVersion = runtime.Version()
+
+	//Using shell commands to get git commit and build info
```

Comment:
This will not work in the field.  You need to generate this during the build.  Probably need to modify the Makefile to change some constant that the code will then look at.  Look at how Docker did this.